### DISCUSSION TITLE
How does implicit messaging work?

### DISCUSSION THREAD

**Discussion by YairLevi** [liftoff-sr/CIPster#41] [Q&A] [answered]
Hi,
I'm looking for someone to answer some questions I have regarding implicit messaging in EIP.
I can't find any official forum or anything so I'm trying to get help here.

In implicit messaging, I never see any code example of someone defining the exact data (instance/class) that they wish to receive, all I see is specifying some pre defined assmebly object.

Can anyone explain to me how this works? where can i define what I want to receive in implicit messaging? and how do I deduce from the response the values?

**Comment by liftoff-sr** [comment, marked as answer]
Implicit messaging is basically the transfer of assemblies using UDP.  You can have as many of them as you want, and there must be advanced agreement and understanding of the layout of assemblies based on documentation from the producing vendor.
